rest/client/methods.go

```diff
@@ -209,6 +210,32 @@ func (c *communicatorImpl) GetVersion(ctx context.Context, taskData TaskData) (*
 	return v, nil
 }
 
+func (c *communicatorImpl) GetProject(ctx context.Context, taskData TaskData) (*model.Project, error) {
+	pp := &model.ParserProject{}
+	info := requestInfo{
+		method:   get,
+		taskData: &taskData,
+		version:  apiVersion1,
+	}
+	info.setTaskPathSuffix("parser_project")
+	resp, err := c.retryRequest(ctx, info, nil)
+	if err != nil {
+		err = errors.Wrapf(err, "failed to get project for task %s", taskData.ID)
+		return nil, err
+	}
+	if resp.StatusCode == http.StatusConflict {
+		return nil, errors.New("conflict; wrong secret")
+	}
+	respBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, errors.Wrap(err, "error reading body")
+	}
+	if err := bson.Unmarshal(respBytes, pp); err != nil {
+		return nil, errors.Wrap(err, "error unmarshalling bson into parser project")
+	}
+	return model.TranslateProject(pp)
```

Comment:
can we push lines 233-236 into a function in the models' package? This will make it easier to refactor later.